hi-ml-cpath/src/health_cpath/datasets/panda_tiles_dataset.py

```diff
@@ -116,7 +116,7 @@ def __len__(self) -> int:
         return len(self.base_dataset)
 
 
-class PandaTilesDatasetWithReturnIndex(DataClassBaseWithReturnIndex, PandaTilesDatasetReturnImageLabel):
+class PandaTilesDatasetWithReturnIndex(DatasetWithReturnIndex, PandaTilesDatasetReturnImageLabel):
     """
     Any dataset used in SSL needs to inherit from DataClassBaseWithReturnIndex as well as VisionData.
```

Comment:
Remaining reference to `DataClassBaseWithReturnIndex` here (and in `tcga_crck_tiles_dataset.py` too and possibly in other files)

Reply:
Thanks, good find, I did not check documentation, but relied on automatic refactoring...